File: app/src/test/java/com/geridea/trentastico/utils/time/WeekTimeTest.java
package com.geridea.trentastico.utils.time;

import org.junit.Test;


/*
 * Created with ♥ by Slava on 20/04/2017.
 */
public class WeekTimeTest {

    @Test
    public void testYearChange() throws Exception {
        WeekTime weekToLoad = new WeekTime(2017, 52);
        WeekTime loadFrom = weekToLoad.copy();

        WeekTime loadTo = loadFrom.copy();
        loadTo.addWeeks(+2);

        new WeekInterval(loadFrom, loadTo);
    }

}

File: app/src/test/java/com/geridea/trentastico/utils/time/WeekTimeTest.kt
package com.geridea.trentastico.utils.time

import org.junit.Test


/*
 * Created with ♥ by Slava on 20/04/2017.
 */
class WeekTimeTest {

    @Test
    @Throws(Exception::class)
    fun testYearChange() {
        val weekToLoad = WeekTime(2017, 52)
        val loadFrom = weekToLoad.copy()

        val loadTo = loadFrom.copy()
        loadTo.addWeeks(+2)

        WeekInterval(loadFrom, loadTo)
    }

}